src/app/layout/components/header/Header.jsx

```diff
@@ -0,0 +1,9 @@
+import * as C from 'components';
+
+const Header = () => (
+  <C.AppBar position="static">
```

Comment:
Roman, please explain what happens here? What is C?

Reply:
It's code styling, it is better when your general components are located in one folder, even if these components are from  another ui-library, since designer can change style for some components and they are supposed to be changed for every page

"import * as something" is very useful, it minimizes development time, there is no necessity to add/remove a components in import when you create feature

"C" is an obvious name for components (like "e" for an event, although I don't really approve of the abbreviated name)

Such an import will not increase the assembly size, since module import is called only once, and everything inside this folder has been used at least once